Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:18.04
```

Comment:
We could use https://github.com/rust-lang/docker-rust here.

Reply:
I think instead of using rust, we should a nodejs image, because
- we have rust and golang, but they're all dev depencies
- resulting service need nodejs, wasm files, ethhashproof binary, they can all run in a nodejs based image

Reply:
Personally I think we should stick to writing our own docker files, using same operating system. In this case if we just stick to ubuntu:18.04 everywhere gives us full power of what do we want to install and how (same package manager, versioning, commands, shell, kernel, etc.) . As well, it gets easier to use the docker images itself, as they are all ubuntu. We could though create our own nodejs, go and rust default images and build on top of them.As these docker images still use Debian which is fine, but I think in our case we should strive to get our base images built in-house.

Reply:
I'm fine to stick on ubuntu, but for a production quality image, only install nodejs. Golang and rust are only _development dependencies_ and should be avoided in result image.

Reply:
Refer to, for example: https://github.com/nearprotocol/nearcore/blob/master/Dockerfile#L46
on how we avoid rust and compilation intermediates into result file (which result in only ~10% in size compare to have rust etc. included)

Reply:
Yep, currently this image is also used for development, that's why we are bundling both ganache and local neard image.